File: PHP/RAM.php
<?php

class RAM {
    private $capacity;  // menampung kapasitas
    private $price;     // menampung harga

    // constructor
    function __construct($capacity, $price) {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // setter and getter
    function setCapacity($capacity) {
        $this->capacity = $capacity;
    }

    function getCapacity() {
        return $this->capacity;
    }

    function setPrice($price) {
        $this->price = $price;
    }

    function getPrice() {
        return $this->price;
    }

    // method
    function printRAM() {
        echo "RAM Capacity      : ". $this->getCapacity(). "<br/>";
        echo "RAM Price         : Rp ". $this->getPrice(). ",-<br/>";
    }
}

?>

File: Python/RAM.py
class RAM:
    __capacity = "" # menampung kapasitas
    __price = 0     # menampung harga

    # constructor
    def __init__(self, capacity = "", price = 0):
        self.__capacity = capacity
        self.__price = price

    # setter and getter
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity
    
    def setPrice(self, price):
        self.__price = price
    
    def getPrice(self):
        return self.__price

    # method
    def printRAM(self):
        print("RAM Capacity     : " + str(self.getCapacity()))
        print("RAM Price        : Rp " + str(self.getPrice()) + ",-")